doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,54 @@
+"""
+===============================================
+Reconstruct dust-covered image using inpainting
```

Comment:
It would be nice if the title could convey that this is a biomedical application.

Reply:
"Restore dust-covered human cornea image using inpainting" this sounds like the cornea itself is covered in dust :smile: but I think it's close. How about "Restore dust-obstructed human cornea image with inpainting"?

Reply:
Sure! That sounds better.